front/app/containers/UsersEditPage/ProfileForm.tsx

```diff
@@ -330,6 +330,7 @@ class ProfileForm extends PureComponent<Props, State> {
             <InputContainer>
               <Input
                 type="text"
+                autocomplete="given-name"
```

Comment:
Just out of interest, why use 'given' (arguably a wrong use of an English term) and 'family' for names, and 'first' and 'last' which seems simpler and more accurate?

Reply:
Because those are the [allowed values](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/autocomplete)!